index.html

```diff
@@ -75,15 +143,33 @@ <h1>Mouse test</h1>
     });
 
     document.addEventListener("mousedown", event => {
-        switch (event.button) {
-            case 0: circle.classList.add("active-left"); break;
-            case 1: circle.classList.add("active-middle"); break;
-            case 2: circle.classList.add("active-right"); break;
-        }
+        circle.classList.add(`active-${event.button}`);
+
+        event.preventDefault();
+
+        const diff = performance.now() - lastClick[event.button];
+        const bad = diff < 200;
+        const name = buttonNames[event.button] || `Button ${event.button}`;
+        console.log(
+            `%c${name} DOWN - Since down: ${diff.toFixed(2)}ms`,
+            bad ? 'background: red; color: #fff' : ''
+        );
+        lastClick[event.button] = performance.now();
     });
 
     document.addEventListener("mouseup", () => {
```

Comment:
Add the parameter here (so we don't use the deprecated `window.event` property):
```suggestion
    document.addEventListener("mouseup", event => {
```